src/CC2Edit/CC2Edit.cpp

```diff
@@ -2537,14 +2537,49 @@ void CC2EditMain::onTestChips2()
         return;
     }
 #ifndef Q_OS_WIN
-    QString winePath = settings.value(QStringLiteral("WineExe")).toString();
-    if (winePath.isEmpty() || !QFile::exists(winePath)) {
-        // Try standard paths
-        winePath = QStandardPaths::findExecutable(QStringLiteral("wine"));
-        if (winePath.isEmpty() || !QFile::exists(winePath)) {
-            QMessageBox::critical(this, tr("Could not find WINE"),
-                    tr("Could not find WINE executable.\n"
-                       "Please configure WINE in the Test Setup dialog."));
+    static auto versionRegex = new QRegExp(QStringLiteral("\\d{10} (?:proton|experimental)-6\\.\\d+-\\d+\n?"));
+    static auto slrSubdir = new QRegExp(QStringLiteral("[a-zA-Z0-9]+_platform_\\d+\\.\\d+\\.\\d+"));
+    QString steamRoot = settings.value(QStringLiteral("SteamRoot")).toString();
+    if (steamRoot.isEmpty() || !QFile::exists(steamRoot)) {
+        // The one which Steam looks for by default
+        steamRoot = QDir::homePath() + QStringLiteral("/.local/share/Steam");
+        if (!QFile::exists(steamRoot)) {
+            QMessageBox::critical(this, tr("Could not find Steam Root"),
+                    tr("Could not find Steam Root directory.\n"
+                       "Please set the Steam Root in the Test Setup dialog."));
+            return;
+        }
+    }
+    // Steam has it's own python version bundled for this, but it also requires injecting library files somehow, it's not worth it

```

Comment:
```suggestion
    // Steam has its own python version bundled for this, but it also requires injecting library files somehow, it's not worth it
```

I may be missing something, but where are we injecting libraries?

Aside from the comment, I'm fine with using the python3 from the user's PATH anyway.

Reply:
What I meant by that comment is that just using Steam's Python doesn't work, as it uses outdated and custom libraries which are dynamically added in via something like the env variable `LD_LIBRARY_PATH`.

Reply:
Ah, I see, that makes more sense...  Perhaps the wording could make this more clear then?  Something like
```c++
    // Steam bundles its own python version with custom libraries that are
    // potentially incompatible with the system versions, so it's not worth
    // using Steam's version here.
```
(or something along those lines)